packages/ui/src/components/App.tsx

```diff
@@ -249,6 +250,13 @@ class _Navigation extends RefreshableComponent<
     this.props.connectedPeersContainer.refreshPeers();
     this.props.networkInfoContainer.refresh();
   }
+
+  async signerConnected() {
```

Comment:
Rename the method name to sth like `signerConnectionStateText()`

Reply:
Will do